src/app/components/absender-verwalten/absender-verwalten.component.html

```diff
@@ -1,15 +1,11 @@
-<div class="col-sm-6">
-  <app-ldap-suche [dragEnabled]="true" (selected)="addToPAL($event)"></app-ldap-suche>
-</div>
-
-<div class="col-sm-6">
+<div class="container">
```

Comment:
Container in bootstrap darf man nicht schachteln. D.h. es darf nur einen Container geben. Komponenten dürfen daher keine Container sein.